### ISSUE TITLE
Does not seem to handle 'ftp' type uri correctly.

### ISSUE THREAD

**Issue by lowks** [johnotander/normalize_url#3]
This is what I get when I try a `ftp` type address:

```
iex(2)> NormalizeUrl.normalize_url("ftp://google.com")
"http://ftp//google.com"
```

**Comment by johno** [comment]
Ah, thanks for reporting. I'll add support for the `ftp` protocol asap.

**Comment by johno** [comment]
Fixed on version `0.0.2` : )
